.github/labeler.yml

```diff
@@ -0,0 +1,55 @@
+# root for top level changes
+root:
+  - '.*'
+
+# any update in README.md
+docs:
+  - README.md
+  - packages/*/src/doc/*
+  - packages/*/README.md
+
+# any file changes with the .test.js extension
+tests:
+  - packages/**/*.test.js
+
+# any change to the workflows file is updated
+workflows:
+  - .github/workflows/*
+
+# For any file changed
+awaitingReview:
+  - '**'
+  - '.*'
+  - '.*/**'
+
+# release any file changed in a package
+release:
+  - packages/*/src/**/*
+
+# add label based on package changed
+charts:
+  - packages/charts/**/*
+components:
+  - packages/components/**/*
+config:
+  - packages/config/**/*
+demo:
+  - packages/demo/**/*
+inventory:
+  - packages/inventory/**/*
+inventory/compliance:
+  - packages/inventory-compliance/**/*
+inventory/general-info:
+  - packages/inventory-general-info/**/*
+inventory/insights:
+  - packages/inventory-insights/**/*
+notifications:
+  - packages/notifications/**/*
+remediations:
+  - packages/remediations/**/*
+sources:
+  - packages/sources/**/*
+translations:
+  - packages/translations/**/*
+utils:
+  - packages/utils/**/*
```

Comment:
:no_good_man:

Reply:
Can you expand on this review? Not really sure what this means 😄

Reply:
:D ok there missing new lines